doc/FAQ.rst

```diff
@@ -48,6 +48,22 @@ This same method is applicable to adjust the range of a color bar. Here
 z_col is the color bar value dimension and is bounded from 0 to 5.
 
 
+**Q: How do I keep all the axes limits consistent/normalized across
+all frames in a HoloMap/all elements in a layout?
+
+**A:** You may use the use the normalization options as specified below:
+
+..code:: python
+    # for consistent limits across frames
+    your_holomap.options(framewise=False)
+
+    # for consistent limits across elements
+    your_layout.options(axiswise=False)
```

Comment:
These are the defaults so it might be nice to expand on this and explain the axiswise=True and framewise=True cases. Particularly for DynamicMaps setting framewise=True means that the ranges are updated on every frame, which is a very frequently asked question.

Reply:
I'll reword it and push to the branch.